python/ql/lib/semmle/python/dataflow/new/internal/DataFlowDispatchPointsTo.qll

```diff
@@ -318,24 +319,30 @@ class DataFlowCallable extends TDataFlowCallable {
   /** Gets the name of this callable. */
   string getName() { none() }
 
-  /** Gets a callable value for this callable, if one exists. */
-  CallableValue getCallableValue() { none() }
-
   /** Gets the underlying library callable, if any. */
   LibraryCallable asLibraryCallable() { this = TLibraryCallable(result) }
 
   Location getLocation() { none() }
 }
 
+abstract class NonLibraryDataFlowCallable extends DataFlowCallable {
+  /** Gets a callable value for this callable, if one exists. */
+  abstract CallableValue getCallableValue();
+
+  abstract CallNode getACall2();
```

Comment:
This needs a better name. Even if it is only internal, it certainly doesn't convey what it's actually supposed to do.